### PR TITLE
Fixing major bugs and splitting cluster_ca and cluster_ca base64

### PR DESCRIPTION
**Pull request by Typositoire**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,13 +8,22 @@ Heavily based on the work of [`linkyard/concourse-helm-resource`][linkyard].
 
 [linkyard]: https://github.com/linkyard/concourse-helm-resource
 
+## IMPORTANT NOTES
+
+- Version 1.21.0 to 1.24.2 seems to be broken for certain uses cases.
```
Comment (avanier) [inline README.md:13]: Needs more information on what case of use-cases we're talking about.

File: README.md [hunk 2]
```diff
@@ -8,13 +8,22 @@ Heavily based on the work of [`linkyard/concourse-helm-resource`][linkyard].
...
+- Version 1.21.0 to 1.24.2 seems to be missing helm diff plugin due to the use of HELM_PLUGINS environment variable
```
Comment (avanier) [inline README.md:14]: Should expand on why. This is actually interesting.

File: README.md [hunk 3]
```diff
@@ -8,13 +8,22 @@ Heavily based on the work of [`linkyard/concourse-helm-resource`][linkyard].
...
+- Feel free to add to this list
```
Comment (avanier) [inline README.md:15]: Moar

File: assets/common.sh [hunk 4]
```diff
@@ -39,10 +40,15 @@ setup_kubernetes() {
         kubectl config set-cluster default --server=$cluster_url --insecure-skip-tls-verify=true
       else
         ca_path="/root/.kube/ca.pem"
-        if ! echo "$cluster_ca" | base64 -d > $ca_path; then
-          echo "info: assuming unencoded cluster CA cert" >/dev/stderr
+        if [[ ! -z "$cluster_ca_base64" ]]; then
```
Comment (avanier) [inline assets/common.sh:43]:
```suggestion
        if [[ -n "$cluster_ca_base64" ]]; then

File: assets/common.sh [hunk 5]
```diff
@@ -39,10 +40,15 @@ setup_kubernetes() {
...
+          echo "$cluster_ca_base64" | base64 -d > $ca_path
+        elif [[ ! -z "$cluster_ca" ]]; then
```
Comment (avanier) [inline assets/common.sh:45]:
```suggestion
        elif [[ -n "$cluster_ca" ]]; then
```

**Review by avanier — CHANGES_REQUESTED** [review]

**Review by marionbrioche — APPROVED** [review]

**Comment by Typositoire** [comment]
Concourse: Version bump found.

File: assets/common.sh (hunk 4 above)
Comment (Typositoire) [reply assets/common.sh:43]: There's a reason for using `-z` that I can't remember. Just trust me :p

File: assets/common.sh (hunk 5 above)
Comment (Typositoire) [reply assets/common.sh:45]: There's a reason for using `-z` that I can't remember. Just trust me :p

**Pull request merged**